cartography/data/jobs/cleanup/gcp_pubsub_topics_cleanup.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
once the topic & subscriptions relationship is established, we should update this and the subscriptions cleanup script